.gitignore

```diff
@@ -217,3 +217,4 @@ ModelManifest.xml
 *.dll
 /published/TLCGen.Setup.msi
 /published/2019-06-27-TLCGen_v0.5.2.0.zip
+TLCGen/TLCGen.csproj
```

Comment:
Ha Dick,
dit lijkt me niet handig voor in de master branch; het project wijzigt regelmatig wegens toevoegingen etc.
Kun je dat nog verwijderen? Vervolgens voer ik de merge uit.
Groet, Menno
ps. je kunt bij een commit ook slechts een keuze uit files committen